### PR TITLE
Set up docker for dev environment

### PR DESCRIPTION
**Pull request by AndersonQ**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -64,6 +64,7 @@ group :development, :test do
 end
 
 group :development do
+  gem 'therubyracer'
```
Comment (giubueno) [inline Gemfile]: Shouldn't we install it directly in our Docker image? I mean, install the NodeJS there directly instead of adding this here?

Comment (AndersonQ) [reply Gemfile]: Well, that way we make the application more independent of the platform it's running on. However I don't mind, what ever makes more cense for a ruby application

Comment (giubueno) [reply Gemfile]: I am not against the Docker Anderson. I think we could have the real NodeJS instead of this therubyracer.

Comment (giubueno) [reply Gemfile]: You can install the NodeJS directly in your docker image.

**Pull request merged**